main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/app-operator/flag"
+	"github.com/giantswarm/app-operator/server"
+	"github.com/giantswarm/app-operator/service"
+)
+
+const (
+	notAvailable = "n/a"
+)
+
+var (
+	description = "The app-operator deploys catalogs' app by look up to appCatalog CR."
```

Comment:
⬇️ Suggested edit.

`The app-operator manages apps in Kubernetes clusters.`

We use a generic description here and the CRD the operator watches may change in the future.